src/status_im2/contexts/onboarding/enable_notifications/style.cljs

```diff
@@ -6,9 +6,8 @@
 
 (defn page-container
   [insets]
-  {:flex             1
-   :padding-top      (:top insets)
-   :background-color colors/neutral-80-opa-80-blur})
```

Comment:
We don't need background color anymore because we now use the screen behind as the background